scripts/firedrake-install

```diff
@@ -492,15 +491,13 @@ if options["petsc_int_type"] == "int32":
     # Serial mesh partitioner
     petsc_options.add("--download-chaco")
     if not options["minimal_petsc"]:
-        # AMG
-        petsc_options.add("--download-ml")
```

Comment:
Why is it necessary to remove this line?

Reply:
When I tried to install with ml, I got the following error in my `firedrake-install.log`

*******************************************************************************
             UNABLE to CONFIGURE with GIVEN OPTIONS    (see configure.log for details):
    -------------------------------------------------------------------------------
    Cannot use ml with complex numbers it is not coded for this capability
    *******************************************************************************
Would it be better to still install ml if real PETSc is built?

Reply:
Yes, It's the same procedure as for hypre, you take the configure option out of the set in the complex case.

Reply:
Okay, I pushed a fix